libs/@local/hash-isomorphic-utils/src/data-types.ts

```diff
@@ -38,7 +38,7 @@ const createFormattedParts = ({
 
 export const formatDataValue = (
   value: JsonValue,
-  schema: ValueConstraint | null,
+  schema?: DataType | SimpleValueSchema,
```

Comment:
What's the motivation for changing this?

The intention of an explicit `null` is to avoid people just calling `formatDataValue(value)` and not passing the schema by mistake (which is easy to do).